lib/ansible/playbook/role/__init__.py

```diff
@@ -472,7 +475,7 @@ def add_parent(self, parent_role):
             self._parents.append(parent_role)
 
     def get_parents(self):
```

Comment:
Given that the patch changes the behavior a little, it might be a good idea to document the intent:
```suggestion
    def get_parents(self):
        """Return a copy of the parents roles list."""
```